src/components/Search/SearchInput.jsx

```diff
@@ -12,47 +12,61 @@ import { connect } from 'react-redux';
 import SearchViewHelper from '../../utils/searchViewHelper';
 import SearchResultsList from './SearchResultsList';
 
-const { handleChangeDropdown, handleChangeSearch } = SearchViewHelper;
+const { handleChangeDropdown, handleChangeSearch, handleClickOutsideInput } =
+  SearchViewHelper;
 
-function SearchInput(props) {
-  const { repository, searchResults } = props;
-  const repoList = ['try', 'autoland', 'mozilla-central'];
+const repoList = ['try', 'autoland', 'mozilla-central'];
 
-  return (
-    <Container maxWidth="md">
-      <FormControl sx={{ width: '25%' }}>
-        <InputLabel id="select-repository">repository</InputLabel>
-        <Select
-          value={repository}
-          labelId="select-repository"
-          label="repository"
-        >
-          {repoList.length > 0 &&
-            repoList.map((item) => (
-              <MenuItem
-                id={item}
-                value={item}
-                key={item}
-                onClick={handleChangeDropdown}
-              >
-                {item}
-              </MenuItem>
-            ))}
-        </Select>
-      </FormControl>
-      <TextField
-        label="Search By Revision ID or Author Email"
-        variant="outlined"
-        sx={{ width: '75%' }}
-        onChange={handleChangeSearch}
-      />
-      {searchResults.length > 0 && <SearchResultsList />}
-    </Container>
-  );
+class SearchInput extends Component {
+  componentDidMount() {
+    document.addEventListener('mousedown', handleClickOutsideInput);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('mousedown', handleClickOutsideInput);
+  }
+
```

Comment:
had to add an event listener when component mounts to detect when user clicks outside of input or results